.circleci/ci_build.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
 set -e
-# set -x
+echo $PATH
+export PATH=$PATH:/usr/local/bin
```

Comment:
Maybe limit this for macOS?

Reply:
Actually it's more frustrating than a version. Using Cataline 10.15.2/7 as examples we're **sometimes** missing it which is more frustrating. So in AWS (for example) it's always there. In Circle, depending on the executor it's present. We've found it both present and not.

Reply:
It's ok to add it even if it's already there, but let's limit that to macOS, ok?

Reply:
@chayim BTW if it's a general macOS fix we shouldn't have it in RediSearch but rather in readies/shibumi/defs or something (like the profile.d macOS fix). WDYT?

Reply:
Once there's a pattern, sure.